server/routes/createPlan/checkYourAnswers/checkYourAnswersController.ts

```diff
@@ -93,7 +94,7 @@ export default class CheckYourAnswersController {
       deleteSessionData(req, ['createPlan', id])
       deleteSessionData(req, ['changeStatus', id])
 
-      res.redirect(addressLookup.learningPlan.addGoals(id))
+      res.redirect(`${config.learningPlanUrl}/plan/${id}/induction-created`)
```

Comment:
You see - the `addressLookup` just feels like a completely unnecessary abstraction 🤷‍♂️